setup.py

```diff
@@ -79,6 +79,9 @@
         # dependency of contextvars, which we vendor
         'immutables==0.6',
     ],
+    extras_require={
+        'display': ["ipython"]
```

Comment:
I think `latex` might be a better name for the extras group -- thoughts? I get that just the "display" function is what needs it, but "display" alone doesnt really ring a bell for me idk

Reply:
I agree, `latex` is more mnemonic